### DISCUSSION TITLE
Plotting correlated misorientation axis on specimen reference frame

### DISCUSSION THREAD

**Discussion by YuvalBon** [mtex-toolbox/mtex#611] [Ask Anything]
Hi!

I'm trying to plot the misorientation axis of subgrain boundaries (meaning, boundaries with misorientation of 1 - 10 degrees inside a grain). I would like to show it in the sample reference frame. 
I know I can calculate the _uncorrelated_ misorientation axis in the sample reference frame (using 'calcMisorientation(ebsd)'), or calculate the correlated misorientation axis but in the _crystal_ reference frame (here using 'grain.boundary.misorientation' or 'grain.innerBoundary').     
It is possible to have the _correlated_ MOA on the _sample reference_ frame in MTEX?

Thanks much!
Yuval

**Comment by lfgmorales** [comment]
Hi Yuval



Would something like


https://mtex-toolbox.github.io/HomepageOld/files/doc/GrainMisorientationAnalysis.html


do what you want?



cheers


L


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________

**Reply by YuvalBon** [reply]
Thanks Luiz!

I followed this code and using 'v = axis(ebsd('Forsterite').orientations,oriRef)' I generated pole figures that looks like the example:
![SGB_3_H6915F_2](https://user-images.githubusercontent.com/75444955/101085504-2ff87a00-35b8-11eb-9398-49fbd681a277.jpg)

However, I am not sure I understand what these figures mean. (a) Are these the axis orientations of correlated or uncorrelated pairs of misorientations?, and (b) is that the sample reference frame (similar to a standard CPO pole figure where the X direction is east-west and Z direction is north-south)?  
Best,
Yuval

**Comment by lfgmorales** [comment]
Hi yuval

Sorry, that was part of the old documebtation

Have a look here

https://mtex-toolbox.github.io/EBSDGROD.html

In this case the ebsd.orientations of (in your case) olivine are conbined to the axis of the grain orientation reference deviation that you can calculate after you have grains calculated, essentially transforming the misorientationnaxes from crystal reference frame to sample reference frame. The sample reference frame is of course the same you setup when importing the data

Does this make sense?

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 3 Dec 2020, at 21:48, YuvalBon <notifications@github.com> wrote:

﻿

Thanks Luiz!

I followed this code and using 'v = axis(ebsd('Forsterite').orientations,oriRef)' I generated pole figures that looks like the example:
[SGB_3_H6915F_2]<https://user-images.githubusercontent.com/75444955/101085504-2ff87a00-35b8-11eb-9398-49fbd681a277.jpg>

However, I am not sure I understand what these figures mean. (a) Are these the axis orientations of correlated or uncorrelated pairs of misorientations?, and (b) is that the sample reference frame (similar to a standard CPO pole figure where the X direction is east-west and Z direction is north-south)?
Best,
Yuval

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/611#discussioncomment-146720>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN4TPHPPMVZOAYYM4ILSS72Q3ANCNFSM4UMLGAUA>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/611#discussioncomment-146720", "url": "https://github.com/mtex-toolbox/mtex/discussions/611#discussioncomment-146720", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by YuvalBon** [reply]
Yes! I believe this is exactly what I was looking for. Many thanks!

Yuval
